tests/storage/test_event_push_actions.py

```diff
@@ -24,15 +24,14 @@
 
 USER_ID = "@user:example.com"
 
-PlAIN_NOTIF = ["notify", {"set_tweak": "highlight", "value": False}]
```

Comment:
Drive-by grammar comment but I think the "l" here needs to be capitalized?

Reply:
Heh, definitely true -- but we're removing this so no need to make a change!